go.mod

```diff
@@ -12,6 +12,7 @@ require (
 
 require (
 	cloud.google.com/go v0.46.3
+	github.com/AlekSi/gocov-xml v1.0.0 // indirect
```

Comment:
this is invalid, as go mod tidy removes this. it's not used by the project per se and should not be in the go.mod